packages/apps/src/HelpWidget.tsx

```diff
@@ -0,0 +1,116 @@
+// Copyright 2017-2020 @polkadot/react-components authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import React, { useState } from 'react';
+import * as moment from 'moment';
+import styled from 'styled-components';
+
+interface Props {
+  className?: string;
+}
+
+const HelpWidget = ({ className }: Props) => {
+  const [visible, setVisible] = useState(false);
+
+  const handleClick = () => {
+    !visible ? setVisible(true) : setVisible(false);
+  };
+
+  return (
+    <div className={className}>
+      {visible &&
+          <div className={'help-menu'}>
```

Comment:
This should probably use the `Menu` component, but I didn't want to tinker with it before the Upload site PR